src/doc/ESMF_install.tex

```diff
@@ -1322,6 +1329,7 @@ \subsection{Installing the ESMF}
 \item {\tt ESMF\_INSTALL\_MODDIR} -- where to install Fortran module files.
 \item {\tt ESMF\_INSTALL\_BINDIR} -- where to install application files.
 \item {\tt ESMF\_INSTALL\_DOCDIR} -- where to install documentation files.
+\item {\tt ESMF\_INSTALL\_CMAKEDIR} -- where to install cmake module files.
```

Comment:
None of the other items have a period (.) at the end of sentence. Maybe all should get one? Either way fine with me, just should be consistent.

Reply:
I could have sworn that there was no period when I reviewed this... but all I can say is that I definitely need (better) glasses! This looks good as is!